cupboard_app/views.py

```diff
@@ -698,3 +702,69 @@ def list(self, request: Request) -> Response:
         all_measurements = get_all_measurements()
         serializer = MeasurementSerializer(all_measurements, many=True)
         return Response(serializer.data)
+
+
+@extend_schema(tags=['CustomIngredients'])
+class CustomIngredientsViewSet(viewsets.ViewSet):
+    MISSING_USER_INFO = 'User missing. Unable to create new custom ingredient.'
```

Comment:
This missing info message should instead be for if ingredient and type is missing in the body